tests/integration/test___init__run_migrate.py

```diff
@@ -85,6 +85,7 @@ async def test_run_migrate(
 
     | Level | Path | Navlink |
     | -- | -- | -- |
+    | 1 | https-canonical-com | [Canonical](https://canonical.com) |
```

Comment:
We probably also should have a case with a more complex url that contains query strings or anchors

Reply:
Since for migration the link is effectively skipped based on the `TableRow.is_external` function, do you think it would be better to add those to test for `is_external`? Anchors and query strings are indeed a good case for which the path creation needs to be updated

Reply:
I have added a few more tests to `is_external` and the algorithm for creating the path